src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MultipleStringLiteralsCheck.java

```diff
@@ -70,6 +70,25 @@
  * <pre>
  * &lt;module name=&quot;MultipleStringLiterals&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *  public class MyClass {
+ *      String a = "StringContents";
+ *
+ *      public void myTest() {
+ *          String a1 = "StringContents"; // violation
+ *          String a2 = "DoubleString" + "DoubleString"; // violation
+ *          String a3 = "SingleString"; // OK
+ *          String a4 = ", " + ", " + ", "; // violation
+ *      }
+ *
+ *     @SuppressWarnings("unchecked")
+ *     public void myTest1(){}
+ *
+ *     @SuppressWarnings("unchecked") // OK
```

Comment:
please put annotation on class and on myTest method ot make example more compact

Reply:
This class s not done, please do

Reply:
I wonder whether "annotation" refers to reasons for "violation", or refer to Annotation in Java

Reply:
Please remove myTest1(), and myTest2().
And place annotations on other places.

Reply:
> Please remove myTest1(), and myTest2(). And place annotations like `SuppressWarnings("unchecked")` on other places.

I wonder if this one meets the demand:
```java
public class MyClass {
  String a = "StringContents";
  static final String a1 = "unchecked"
  @SuppressWarnings("unchecked") // violation, "unchecked" occurs twice
  public void myTest() {
    String a2 = "StringContents"; // violation, "StringContents" occurs twice
    String a3 = "DoubleString" + "DoubleString"; // violation, "DoubleString" occurs twice
    String a4 = "SingleString"; // OK
    String a5 = ", " + ", " + ", "; // violation, ", " occurs three times
  }
}
```

Reply:
Yes

Reply:
Done